consensus/state.go

```diff
@@ -880,21 +912,11 @@ func (cs *State) handleMsg(mi msgInfo) {
 		// TODO: If rs.Height == vote.Height && rs.Round < vote.Round,
 		// the peer is sending us CatchupCommit precommits.
 		// We could make note of this and help filter in broadcastHasVoteMessage().
+		return err
 
 	default:
 		cs.Logger.Error("unknown msg type", "type", fmt.Sprintf("%T", msg))
-		return
-	}
-
-	if err != nil {
-		cs.Logger.Error(
```

Comment:
Was a minor cosmetic change to move this log up to the caller, maybe better to leave it here